rm1/src/components/Counter.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+
+//react class component
+export default class Counter extends Component {
+    constructor(){
+        super()
+    }
+
+state = {
```

Comment:
Since you're already using class properties for your state and methods consider omitting "constructor" entirely.
https://medium.com/front-end-weekly/class-properties-with-react-dbc9b33e901c
Constructor was used for binding "this" context for methods since there was no way to use arrow functions.

Reply:
Thanks, resolved